lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-infra_manager-template.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Openstack_InfraManager_Template < MiqAeServiceTemplateInfra
```

Comment:
Yes, that one is bad as well.  

The active_record model for cloud has:
`class ManageIQ::Providers::Openstack::CloudManager::Template < ManageIQ::Providers::CloudManager::Template`
so the automate service model should derive from:
`MiqAeServiceManageIQ_Providers_CloudManager_Template`